lib/GetAlbumArt.js

```diff
@@ -0,0 +1,17 @@
+const searchLyrics = require('./SearchLyrics');
+const getAlbumArtFromPath = require('./GetAlbumArtFromPath');
+
+/**
+ * @param {{apiKey: string, title: string, artist: string, optimizeQuery: boolean}} options
+ */
+module.exports = async function(options) {
+	try {
+		let results = await searchLyrics(options);
+		if (!results) return null;
+		let artUrl = await getAlbumArtFromPath(results[0].path);
+		if (artUrl.length === 0 || artUrl.length > 4000) return null;
```

Comment:
This second condition in this check is redundant. It was present in the getLyrics method to handle an edge case where Genius returns awfully long stories out of nowhere (Actual stories involving kings & princess & stuff)